worlds/metroidprime/__init__.py

```diff
@@ -66,22 +67,26 @@ def create_items(self) -> None:
                 continue
             elif i == "Missile Expansion":
                 for j in range(0, 8):
-                    self.multiworld.itempool += [self.create_item('Missile Expansion', True)]
+                    self.multiworld.itempool += [
```

Comment:
Formatter automatically changed these 🤷Let me know if style-wise you want me to configure it so it goes back to how it was. I'm just running the default PEP8 one.